tests/helix/send-to-helix-inner.proj

```diff
@@ -8,13 +8,15 @@
 
     <IncludeDotNetCli>true</IncludeDotNetCli>
     <DotNetCliPackageType>sdk</DotNetCliPackageType>
+    <DotNetCliVersion>8.0.401</DotNetCliVersion>
```

Comment:
How will this work for template tests for 9.0 templates? won't they require a 9.0 SDK?

Reply:
Still going through a first pass to get tests passing :D

Reply:
This will require us to bundle the sdk ourselves, because arcade doesn't seem to support additional runtimes.
To confirm that this works I think this PR should include a couple of tests where the test project targets net9.0 (best in workload tests).